shotgun_api3/shotgun.py

```diff
@@ -1473,10 +1497,13 @@ def update_project_last_accessed(self, project, user=None):
         record = self._call_rpc("update_project_last_accessed_by_current_user", params)
         result = self._parse_records(record)[0]
 
-
-    def _get_session_token(self):
-        """Hack to authenticate in order to download protected content
-        like Attachments
+    def get_session_token(self):
+        """
+        Get the session token associated with the current session.
+        If a session token has already been established, the token for this 
```

Comment:
"the token for this is returned".
Sounds odd. What is this "this" you are speaking of?
"it is returned" feels better IMHO.